scripts/migrate_registration_logs.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from modularodm import Q
+
+from website.models import NodeLog
+from website.app import init_app
+
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_targets():
+    logs = NodeLog.find(
+            Q('action', 'eq', 'embargo_approved') |
+            Q('action', 'eq', 'embargo_initiated') |
+            Q('action', 'eq', 'retraction_approved') |
+            Q('action', 'eq', 'retraction_initiated') |
+            Q('action', 'eq', 'registration_initiated')
+    )
+    return logs
+
+
+def migrate_log(logs):
+    for log in logs:
+        if log.node.registered_from:
+            log.params['node'] = log.node.registered_from_id
+        else:
+            log.params['node'] = log.node.logs[0].node._id
```

Comment:
@icereval some of the registrations that logs belong to have already been deleted. There is not a registered_from_id field. In that case, we find the first log of that registration which is the registered from node's creation log to get the original node's id.